src/data-mgt/node/utils/transform.js

```diff
@@ -178,22 +205,24 @@ const transform = {
 
   transformMeasurement: async (measurement) => {
     try {
-      let responseFromTransformFieldValues =
-        await transform.trasformFieldValues(measurement.field8);
+      const deviceCategory = measurement.field9
+        ? measurement.field9
```

Comment:
all that do not have category have been labelled as lowcost. This needs review in future PRs.